lib/pushmi_pullyu/preservation_queue.rb

```diff
@@ -61,11 +61,11 @@ def next_item
   end
 
   def wait_next_item
-    loop do
-      element = next_item
-      return element if element.present?
-      sleep @poll_interval
-    end
+    # loop do
```

Comment:
We need a way to close this loop down when a interrupt is given then? Some global flag that calls break if true or something?